templates/ObjectSchema.stencil

```diff
@@ -0,0 +1,20 @@
+// --------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// --------------------------------------------------------------------------
+
+
+import AzureCore
+import Foundation
+
+public struct {{ object.language.default.name }} {
+    {% for property in object.properties %}
+    /* {{ property.language.default.description }} */
+    public let {{ property.language.default.name | lowercase }} : {{ String(property.schema.type) }}
```

Comment:
Cut `| lowercase` and leave the casing to the namer. use `property.name` and `property.description`.

The comment needs to be a swift doc-comment (either /// or /** */) to be picked up by the Xcode help system and Jazzy doc generation.

If the description is nil, then no comment should be output. Looking at this I would think it would still output `/* */`